membersrvc/ca/ca.go

```diff
@@ -33,7 +33,7 @@ import (
 
 	"github.com/hyperledger/fabric/core/crypto/primitives"
 	pb "github.com/hyperledger/fabric/membersrvc/protos"
-	_ "github.com/mattn/go-sqlite3"
+	_ "github.com/mattn/go-sqlite3" // TODO: not use a blank import!
```

Comment:
Do we know why this was here and why it is now marked TODO?

Blank imports are generally used for one purpose to my knowledge: to force the side-effects of running the modules func init().  

So, my first question is: Did a human or the tooling decide to add that comment.  If a human, what was the rationale?

Second question: is the side-effect inducing behavior desired, or not?  If desired, this needs to stay the way it was.  If it was not desired, we should just remove the import outright.